Admin/CommentAdmin.php

```diff
@@ -146,9 +146,9 @@ public function postUpdate($object)
     }
 
     /**
-     * @param ManagerInterface $commentManager
+     * @param CommentManagerInterface $commentManager
      */
-    public function setCommentManager(ManagerInterface $commentManager)
+    public function setCommentManager(CommentManagerInterface $commentManager)
```

Comment:
Because we call the ``updateCommentsCount`` method in https://github.com/core23/SonataNewsBundle/blob/phpdoc/Admin/CommentAdmin.php#L121

Same reason for constructor change on ``RecentCommentsBlockService`` and ``RecentPostsBlockService``.